src/relay/pass/gradient.cc

```diff
@@ -85,10 +85,10 @@ using ADValue = std::shared_ptr<ADValueNode>;
 
 /*! \brief AD over a program which generates a tensor output. */
```

Comment:
What if the program generates a tuple of tensors output?
`ADFunction` and `ADTensor` cannot cover this case.